src/pages/CreatePool/index.tsx

```diff
@@ -1,39 +1,44 @@
-import { BigNumberish, ethers } from "ethers"
 import {
   Box,
   Button,
   Container,
   Divider,
+  FormControl,
   IconButton,
   InputAdornment,
+  InputLabel,
   Link,
+  MenuItem,
   Paper,
+  Select,
   Stack,
   TextField,
   ToggleButton,
   ToggleButtonGroup,
   Typography,
   useTheme,
 } from "@mui/material"
-import { ChainId, PoolTypes } from "../../constants"
-import React, { useEffect, useState } from "react"
+import React, { useContext, useEffect, useState } from "react"
+import { getContract, isAddress } from "../../utils"
 
 import ArrowBackIcon from "@mui/icons-material/ArrowBack"
+import { BigNumberish } from "ethers"
 import CircularProgress from "@mui/material/CircularProgress"
 import DeleteForeverIcon from "@mui/icons-material/DeleteForever"
 import ERC20_ABI from "../../constants/abis/erc20.json"
 import { Erc20 } from "../../../types/ethers-contracts/Erc20"
+import { ExpandedPoolsContext } from "../../providers/ExpandedPoolsProvider"
+import { PoolTypes } from "../../constants"
 import ReviewCreatePool from "./CreatePoolDialog"
 import { Link as RouteLink } from "react-router-dom"
-import { getContract } from "../../utils"
 import { useActiveWeb3React } from "../../hooks"
-import { usePoolRegistry } from "../../hooks/useContract"
 import { useTranslation } from "react-i18next"
 
 export enum PoolType {
   UsdMeta = "usdMetapool",
   BtcMeta = "btcMetapool",
   Base = "basepool",
+  Empty = "",
 }
 
 export type ValidationStatus =
```

Comment:
I think this should be an enum instead so that on L231, you won't have to cast the type and just use `ValidationStatus.ERROR`. This would also help us use the enum strings instead.